### DISCUSSION TITLE
KerasNLP is working on APIs that give you pretrained models like BERT

### DISCUSSION THREAD

**Discussion by chenmoneygithub** [keras-team/keras-hub#296] [General]
Hi community, 

This is just an unofficial announcement that we are working on bringing up APIs to load pretrained models like BERT.

A little different from [Huggingface](https://huggingface.co/), which is doing an awesome job shipping a wide range of NLP models, we are going to focus on a small set of selected models (mainly selected due to its popularity), including BERT, XLM-Roberta, GPT-2 and etc. We will make sure not only shipping the API, but also readable code and building blocks, so that you can easily understand and customize the model. We will also provide training templates that demonstrate the pretraining workflow, which you can customize and run on your desktop, or cloud instances. 

Please stayed tuned! 

Best,
Chen

**Comment by jbischof** [comment]
See https://github.com/keras-team/keras-nlp/issues/297 for our proposal.
